index.js

```diff
@@ -69,6 +69,9 @@ function promisedHandlebars (Handlebars, options) {
   // Re-register all built-in-helpers to ensure that their methods are wrapped
   engine.registerHelper(engine.helpers)
 
+  // Preserve synchronous 'compile' function
+  engine.compileSync = engine.compile.bind({})
```

Comment:
This is (https://github.com/wycats/handlebars.js/blob/master/lib/handlebars.js#L16)[correct], I would nevertheless bind it to the engine, just in case this is changing in the future.

However, see my comment below.